core/core_integrations/telegram.js

```diff
@@ -1,15 +1,7 @@
-var options = {
-    // Used by the monkey patch to node-telegram-bot-api
-    stopPolling: true
-},
-    TelegramBot = require.safe('node-telegram-bot-api'),
-    bot = null,
+var bot = null,
+    api = null,
     shim = require('../shim.js'),
```

Comment:
@mrkno So, is there another way to do this now? The module is in Kassy's package.json, so perhaps .safe is not even required here?